_functions.sh

```diff
@@ -704,14 +704,23 @@ initialize_product_settings() {
               env_var "DEPLOYMENT_ORACLE_DEFAULT_VERSION" "12cR2_plf" # Default version of the oracle server to use
               env_var "DEPLOYMENT_SQLSERVER_DEFAULT_VERSION" "2017-CU2" # Default version of the sqlserver server to use
 
-            else 
+          elif [[ "${PRODUCT_VERSION}" =~ ^(5.3) ]]; then
+              env_var "DEPLOYMENT_ES_IMAGE_VERSION" "1.2.0"
```

Comment:
Starting with 5.2 the version to use is 1.2.1

```suggestion
              env_var "DEPLOYMENT_ES_IMAGE_VERSION" "1.2.1"
```

please update also eXo 5.2 case